.github/workflows/workflow.yaml

```diff
@@ -51,6 +54,48 @@ jobs:
             ./bazel-ci-bin/tests/run-tests
             bazel coverage //tests/...
             '
+
+  test-nixpkgs-cross:
+    name: Build & Test - Cross
```

Comment:
Could you add this to the [mergify config](https://github.com/tweag/rules_haskell/blob/master/.mergify.yml#L3). You can use the [mergify dashboard](https://dashboard.mergify.io/github/tweag/queues) to make sure the config is valid.